src/components/atoms/BubblesLoader.tsx

```diff
@@ -0,0 +1,49 @@
+import { View } from 'native-base'
+import React from 'react'
+import Animated from 'react-native-reanimated'
+
+import { useBubblesLoader } from '~hooks/loaders'
+
+export type BubblesLoaderType = {
+  color?: string
+  size?: number
+}
+
+const FullCircle = ({ size = 10, color = 'black' }: BubblesLoaderType): JSX.Element => (
+  <View style={{ backgroundColor: color, borderRadius: size / 2, width: size, height: size }} />
+)
+
+export const BubblesLoader = ({ size = 40, color = 'black' }: BubblesLoaderType): JSX.Element => {
+  const { animatedHeight, animatedRotate, animatedWidth } = useBubblesLoader()
+
+  const containerStyle = {
+    width: size,
+    height: size,
+    alignItems: 'center',
+    justifyContent: 'center',
+  } as const
+
+  const rowStyle = {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  } as const
+
+  const absoluteStyle = {
+    justifyContent: 'space-between',
+    position: 'absolute',
+  } as const
+
+  return (
+    <Animated.View style={[containerStyle, animatedRotate]}>
```

Comment:
Could be `Animated.Center`

Reply:
`Property 'Center' does not exist on type 'typeof Animated'`